### PR TITLE
Reworked the test node guide to use Symbol Bootstrap

### PR DESCRIPTION
**Pull request by segfaultxavi**

### CODE REVIEW

**Review by rg911 — CHANGES_REQUESTED** [review]
It looks to me the doc is missing a voting node instruction?

**Comment by segfaultxavi** [comment]
True. Sorry.

File: source/guides/network/running-a-test-net-node.rst [hunk 1]
```diff
@@ -9,154 +9,196 @@
...
+
+.. note:: Keep you Secret Keys secret at all times!
+
+Use the information in the ``signing`` section to access the node's account.
+
+***************************
+Providing funds to the node
+***************************
+
+Before the node can begin harvesting or voting (see below) it needs a **minimum amount of funds**. In the ``testnet`` test environment you can use |codename|'s **Faucet** to provide these funds.
```
Comment (rg911) [inline source/guides/network/running-a-test-net-node.rst:109]: minimum amount (probably listed in the harvesting section in the doc):
- harvesting: 10k
- voting: 3 million  
Should it be mentioned here ??

File: source/guides/network/running-a-test-net-node.rst [hunk 2]
```diff
@@ -9,154 +9,196 @@
...
 
 .. code-block:: bash
 
-     Name                       Command                    State   Ports
-     ------------------------------------------------------------------------------------
-     peerassembly_peer-node_1   bash -c /bin-mount/wait    Up      0.0.0.0:7900->7900/tcp
+    symbol-bootstrap link
 
-The node is running if the state for ``peerassembly_peer-node_1`` is set to "Up".
+This creates the required :ref:`VrfKeyLinkTransaction <vrf-key-link-transaction>` and submits it to the network. If it succeeds (it might take some seconds, as the transaction needs to be approved and incorporated into the blockchain) your new node is ready to harvest.
```
Comment (rg911) [inline source/guides/network/running-a-test-net-node.rst:134]: not sure it is good to use `incorporated` here? 
Shall we say `Once the transaction gets confirmed` instead?

File: source/guides/network/running-a-test-net-node.rst (hunk 1 above)
Comment (fboucquez) [reply source/guides/network/running-a-test-net-node.rst:109]: @Wayonb should we state the actual testnet number for harvesting and voting in the guide?

File: source/guides/network/running-a-test-net-node.rst (hunk 2 above)
Comment (fboucquez) [reply source/guides/network/running-a-test-net-node.rst:134]: I would prefer `confirmed` too.

**Comment by fboucquez** [comment]
It looks really good @segfaultxavi ! great work, same comments as Steve

File: source/guides/network/running-a-test-net-node.rst (hunk 1 above)
Comment (segfaultxavi) [reply source/guides/network/running-a-test-net-node.rst:109]: I'd rather not have all this data spread over all the docs, so we don't need to update every page when the data changes.
Plus, this quantity is configurable, right?
So, yeah, I prefer mentioning it only in one place (the harvesting guide, probably).

File: source/guides/network/running-a-test-net-node.rst (hunk 2 above)
Comment (segfaultxavi) [reply source/guides/network/running-a-test-net-node.rst:134]: Hmmm, yeah, poor choice of words on my side :)

File: source/guides/network/running-a-test-net-node.rst (hunk 1 above)
Comment (fboucquez) [reply source/guides/network/running-a-test-net-node.rst:109]: Fair enough... I also think those number could be tuned

**Review by fboucquez — APPROVED** [review]

**Pull request merged**